src/index.json

```
@@ -3146,4 +3146,4 @@
         ]
     },
     "formatVersion": "1"
-}
\ No newline at end of file
+}
```

```suggestion
}
```

remove the newline in the end. It's added by Github.